### ISSUE TITLE
Manifest localhost

### ISSUE THREAD

**Issue by yevheniiIvanise** [manuelbieh/react-ssr-setup#41]
Hi. You generate manifest.json under build/client/static with the following content:
```
{
  "bundle.css": "http://localhost:8501/static/bundle.css",
  "bundle.js": "http://localhost:8501/static/bundle.js",
  "vendor.js": "http://localhost:8501/static/vendor.4e558245.chunk.js",
  "assets/react.svg": "http://localhost:8501/static/assets/react.9a28da9f.svg"
}
```
Could you tell where in the code you add localhost prefix for file aliases? I mean http://localhost:8501/static/assets/react.9a28da9f.svg?

**Comment by manuelbieh** [comment]
Ah, no, it gets set in the start script of the devServer so we can react dynamically to different ports:

https://github.com/manuelbieh/react-ssr-setup/blob/master/scripts/start.js#L32-L38

Maybe I should add a new env variable like `DEVELOPMENT_HOST`

**Comment by manuelbieh** [comment]
I knew I had something in my mind … https://github.com/manuelbieh/react-ssr-setup/issues/13 😉

**Comment by yevheniiIvanise** [comment]
Yeah, thank you. I knew it was somewhere there. But it didn't work for me because I made an assignment after I pass configs for webpack function call. Could you also tell please why you at all serve static files onto separate port(8501 instead of 8500)?

**Comment by manuelbieh** [comment]
8500 is the port for the Node.js app that's doing the server side rendering. 8501 (or more specific: `process.env.PORT + 1`) is the port the Webpack Dev Server is serving its files on. That's necessary for hot reloading. 

You could make the Webpack Dev Server a middleware of the SSR app but since the SSR app gets restarted on file changes to serve the most recent version or your app on a hard reload it would at the same time kill and restart the Webpack Dev Server which in turn would make the client side hot reloading fail.

So I'm starting the Webpack Dev Server on Port 8501 (and never stop or restart it during development) and the Server Side Render app on Port 8500 which gets restarted on every file change to make sure you always see the most recent version of your app both on the client and on the server.
